downloader_app/tasks.py

```diff
@@ -23,12 +29,12 @@ def download_source(source, dates, point1, point2, opt=False):
     """
 
     try:
-        logger.info("Capturando {} {}".format(source, dates))
-        td.download_tiffs(source, dates, point1, point2, opt)
+        logger.info("Fetch {} {}".format(source, dates))
+        td(source, dates, point1, point2, opt)
 
     except Exception as e:
         logger.error(
-            "Error capturando from {} at {} error: {}".format(source, dates, e)
+            "Error fetch from {} at {} error: {}".format(source, dates, e)
```

Comment:
you can simplify it using the standard EE for error and II for information, example:

`"[EE] fetching from {} at {} error: {}`